### PR TITLE
Microphone Soft Switch Test Novacustom

### PR DESCRIPTION
**Pull request by wiktormowinski**

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 1]
```diff
@@ -2264,6 +2264,68 @@ can reset the EC.
...
+
+1. Proceed with the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Go to 'Settings -> System -> Sound
```
Comment (mkopec) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:2319]:
```suggestion
1. Go to 'Settings -> System -> Sound'
```

Comment (mkopec) [reply docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:2319]: Or maybe you meant to use ` (code block) here, I'm not sure

Comment (wiktormowinski) [reply docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md]: exactly and let's stick to that: https://github.com/Dasharo/docs/pull/876/commits/a39b2eb6fbb55f718e6c0c079e3f84d41f401fd9

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 2]
```diff
@@ -2264,6 +2264,68 @@ can reset the EC.
...
+
+1. Proceed with the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Go to 'Settings -> Sound
```
[the remark this answers is not in the document]
Comment (wiktormowinski) [reply docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md]: mb fix here: https://github.com/Dasharo/docs/pull/876/commits/a39b2eb6fbb55f718e6c0c079e3f84d41f401fd9

**Review by mkopec — APPROVED** [review]

**Pull request merged**